src/rules/normal_mean_variance/var.jl

```diff
@@ -5,19 +5,19 @@
     m_out_mean        = mean(m_out)
     m_μ_mean, m_μ_cov = mean_cov(m_μ)
     
-    return ContinuousUnivariateLogPdf(DomainSets.HalfLine(), (x) -> -log(m_μ_cov + x) / 2 - 1 / (2x) * (m_out_mean - m_μ_mean) ^ 2)
+    return ContinuousUnivariateLogPdf(DomainSets.HalfLine(), (x) -> -log(m_μ_cov + x) / 2 - 1 / (2*(x + m_μ_cov)) * (m_out_mean - m_μ_mean) ^ 2)
 end
 
 @rule NormalMeanVariance(:v, Marginalisation) (m_out::UnivariateNormalDistributionsFamily, m_μ::PointMass) = begin 
     m_out_mean, m_out_cov = mean_cov(m_out)
     m_μ_mean              = mean(m_μ)
 
-    return ContinuousUnivariateLogPdf(DomainSets.HalfLine(), (x) -> -log(m_out_cov + x) / 2 - 1 / (2x) * (m_out_mean - m_μ_mean) ^ 2)
+    return ContinuousUnivariateLogPdf(DomainSets.HalfLine(), (x) -> -log(m_out_cov + x) / 2 - 1 / (2*(x + m_out_cov)) * (m_out_mean - m_μ_mean) ^ 2)
 end
 
 @rule NormalMeanVariance(:v, Marginalisation) (m_out::UnivariateNormalDistributionsFamily, m_μ::UnivariateNormalDistributionsFamily) = begin 
     m_out_mean, m_out_cov = mean_cov(m_out)
     m_μ_mean, m_μ_cov     = mean_cov(m_μ)
 
-    return ContinuousUnivariateLogPdf(DomainSets.HalfLine(), (x) -> -log(m_out_cov + m_μ_cov + x) / 2 - 1 / (2x) * (m_out_mean - m_μ_mean) ^ 2)
+    return ContinuousUnivariateLogPdf(DomainSets.HalfLine(), (x) -> -log(m_out_cov + m_μ_cov + x) / 2 - 1 / (2*(m_out_cov + m_μ_cov + x)) * (m_out_mean - m_μ_mean) ^ 2)
```

Comment:
Do we have derivations for this one? I am not sure whether an extension is so straightforward.

Reply:
![image](https://user-images.githubusercontent.com/6557701/165247864-038b98bb-904b-44b4-9e15-a5165724cba3.jpeg)